src/picknik_ur_site_config/objectives/open_door_lever_handle_MTC_config.yaml

```diff
@@ -4,7 +4,7 @@ SetupMTCGraspAndTwistThenMoveAlongArcPush:
   ik_frame_name: "grasp_link"
 
   # List of controllers to use during the stages that interact with the door.
-  door_opening_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
+  move_along_arc_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
```

Comment:
Same here, ensure the Gazebo configuration overrides this file to not use the admittance controller, as it does not work.